src/main/java/io/aiven/klaw/service/EnvControllerService.java

```diff
@@ -79,20 +80,13 @@ public void updateEnvsStatus() {
 
   private void updateEnvStatusPerEnv(Integer tenantId, Env env) {
     String status;
+    KwClusters kwClusters =
+        manageDatabase.getClusters(env.getType(), tenantId).get(env.getClusterId());
```

Comment:
Probably it's better to have `getClusters` taken `KafkaClustersType` as argument rather than `String` however I would say it is a topic for a separate issue

Reply:
agree, this has to be changed through out, and prefer in a separate issue. Created : https://github.com/aiven/klaw/issues/85